DRODLib/Character.cpp

```diff
@@ -4577,6 +4577,10 @@ bool CCharacter::DidPlayerInput(
 		return false;  //nLastCommand is a false "wait"
 	if (px == CMD_EXEC_COMMAND) {
 		return CueEvents.HasOccurred(CID_CommandKeyPressed); //only way to detect
+	} else if (px == CMD_EXEC_COMMAND_TWO) {
```

Comment:
Nit: don't need the "else", since there's a return in the then-block. I think it reads cleaner this way.